firmware/application/apps/ais_app.hpp

```diff
@@ -72,51 +72,43 @@ struct AISRecentEntry {
 	size_t received_count;
 	int8_t navigational_status;
 
-	AISRecentEntry(
-	) : AISRecentEntry { 0 }
-	{
-	}
+	AISRecentEntry() : AISRecentEntry{0} {}
 
-	AISRecentEntry(
-		const ais::MMSI& mmsi
-	) : mmsi { mmsi },
-		name { },
-		call_sign { },
-		destination { },
-		last_position { },
-		received_count { 0 },
-		navigational_status { -1 }
-	{
-	}
+	AISRecentEntry(const ais::MMSI& mmsi)
```

Comment:
Is there a setting for this indent on the init. There's no reason to indent this at the open (

Reply:
There will be, but I'm not sure what that setting would be